### PR TITLE
Unit tests

### LINKED ISSUE #83: Unit tests
> I would love some unit tests on the project, I didn't need them at the time because I was interactively testing on the printer, so you can consider the ConsoleTest as an integration test suite. The main issue with the unit test design will be what is under test, and how we can properly assert failures. Should we test for specific byte strings on the emitters? Which sections should have test cases built first? Why don't you open an issue where we can discuss the unit tests and add them to the project/wiki for coordination?

From Luke in [here](https://github.com/lukevp/ESC-POS-.NET/pull/81#pullrequestreview-500691111)

### PR DESCRIPTION
**Pull request by igorocampos**
Fixes #83 

@lukevp  I think this is a start, please check it out so a good pattern can be established.

### CODE REVIEW

File: ESCPOS_NET.UnitTest/ESCPOS_NET.UnitTest.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```
Comment (igorocampos) [inline ESCPOS_NET.UnitTest/ESCPOS_NET.UnitTest.csproj:10]:
```suggestion
```

**Review by igorocampos — COMMENTED** [review]
Ops forgot to remove MSTests pack

**Review by lukevp — APPROVED** [review]
Looks great, thanks @igorocampos !

**Comment by igorocampos** [comment]
You're welcome @lukevp

**Pull request merged**